### ISSUE TITLE
Vpilot (VATSIM) microphone chosen in audio panel issue

### ISSUE THREAD

**Issue by Siberier** [flybywiresim/a32nx#7457]
### Aircraft Version

Experimental

### Build info

```json
{
    "built": "2022-08-28T23:16:40+00:00",
    "ref": "refs/heads/experimental",
    "sha": "60052b5f1be49db7b375b75aed804a41a4e378e8",
    "actor": "BlueberryKing",
    "event_name": "manual",
    "pretty_release_name": "experimental:60052b5f"
}
```

### Describe the bug

The problem is  the Vpilot dont change MIC from primary to secondary station when you do it in audiopanel in the cockpit. 

### Expected behavior

The Vpilot has to change active transmitter acording audipanel in the piedistal.

### Steps to reproduce

1. start online via Vpilot
2. set freq on the left COM
3. set freq on the right COM
4. try to change MIC button on audipanel in the sim
5. check Vpilot panel

**Comment by juliansebline** [comment]
Most likely due to my work. I'll have a look

**Comment by globin** [comment]
Did you change it on both sides? Otherwise both are active

**Comment by juliansebline** [comment]
Yes that's true, if you want to talk on both, you have to press the respective button on the second ACP

**Comment by juliansebline** [comment]
It works for me. Keep in mind the second ACP is on VHF1 too.

I suggest you to press the VHF2 CALL button on the ACP 2 to be able to talk on both.

**Comment by juliansebline** [comment]
Hello, will be fixed in #7241 

A workaround is to press the CALL button on the left RMP. By default, the RMP will have no transmit channel.
